### PR TITLE
Add SetFlipVerticallyOnLoad method

### PR DESCRIPTION
**Pull request by jchand99**
- Adds SetFlipVerticallyOnLoad method that wraps stbi_set_flip_vertically_on_load function from stb_image.h
  - Static function within Stbi class

Usage:
```c#
Stbi.SetFlipVerticallyOnLoad(true);
```

### CODE REVIEW

File: stbi-sharp.cs [hunk 1]
```diff
@@ -204,6 +204,13 @@ public static void InfoFromMemory(MemoryStream data, out int width, out int heig
         [DllImport("stbi")]
         unsafe public static extern byte* LoadFromMemory(byte* data, long len, out int width, out int height, out int numChannels, int desiredNumChannels);
 
+        /// <summary>
+        /// flip the image vertically, so the first pixel in the output array is the bottom left
```
Comment (peppy) [inline stbi-sharp.cs]:
```suggestion
        /// Flip the image vertically, so the first pixel in the output array is the bottom left.

```

**Review by peppy — CHANGES_REQUESTED** [review]
Just a tiny comment change for consistency, looks good otherwise!

File: stbi-sharp.cs (hunk 1 above)
Comment (jchand99) [reply stbi-sharp.cs]: Awesome! Thank you!

**Review by peppy — APPROVED** [review]

**Comment by Tom94** [comment]
I am mentally debating whether to expose STBI's stateful API (as done in this PR) or whether to instead add an optional parameter `bool flipVertically = false` to all the loading functions. I have a strong dislike for stateful APIs (just look at OpenGL :p) and STBI currently only has 2 state parameters, where the other one (unpremultiplying of alpha on iPhone pngs) arguably never should be touched.

Any opinions?

**Comment by peppy** [comment]
Given there's only two, and they sound like toggles you'd *probably* only change state of once during execution (in *most* use cases) I think it's fine to be exposed as the native API. But honestly either would work. The argument for doing it as stateful API is one (or two) less extern calls...

Quite neutral though.

**Comment by jchand99** [comment]
When I was writing this change, I also went back and forth a bit on whether I wanted to add a flag to each of the functions, but decided against it because based on the way I was thinking to implement the flag, it added more conditional logic that was duplicated across each of the load methods. Because of this I decided just to expose the native API since it was less code. Also, as @peppy mentioned, the state changes most likely would be changed only once during runtime in *most* cases. I think that having the two states exposed as static methods in the Stbi class is fine, but either way would work and I would be willing to use either option, just personal preference really.

Side note: Ofc, the logic for the flag could be extracted to a function that would be called per load method, but the logic would still be run across each load method.

**Comment by Tom94** [comment]
That's fair. Gonna hold off from doing anything overcomplicated and stick as closely as possible to the native API. Thanks. :)

**Pull request merged**